docs/sources/configuration/_index.md

```diff
@@ -1006,10 +1006,11 @@ pool_config:
   # How quickly a dead client will be removed after it has been detected
   # to disappear. Set this to a value to allow time for a secondary
   # health check to recover the missing client.
-  [remotetimeout: <duration>]
+  # CLI flag: -ingester.client.healthcheck-timeout
+  [remote_timeout: <duration>]
```

Comment:
Should we add the default of 1s ?

Reply:
It should be OK, `pool_config` only exists here. Added in [3c8e584](https://github.com/grafana/loki/pull/5418/commits/3c8e5841167b027ff64bda22d88bd69488935767).